src/be/app.py

```diff
@@ -28,16 +32,52 @@ def static(path: str):
         path=_fe_dir / path,
     )
 
-
 @app.get("/api/v1/status")
 def status():
+    #Use the `bulb` variable to get the properties of the LED strip, such as its name, power status, brightness, and color. Return these properties as an instance of the `Status` class.
+    properties = bulb.get_properties()
+    bulb_name_variable = properties['name'] or "Unknown"
+    power_status = properties['power']
+    brightness = properties['bright']
+    colour = properties['rgb']
+    if bulb_name_variable is None:
+        bulb_name_variable = 'Unknown'
+    print(f"Bulb name: {bulb_name_variable}, Power status: {power_status}, Brightness: {brightness}, Colour: {colour}")
     return Status(
-        bulb_name="TODO",
+        bulb_name= bulb_name_variable,
+        power_status='on',
+        brightness=0,
+        colour=(0xFF, 0xFF, 0xFF)
     )
 
 @app.post("/api/v1/turn_on")
 def turn_on():
-    # TODO turn bulb on here
-    return "Hello from the server! implement this function"
+    bulb.turn_on() 
+    return bulb.turn_on()
+
+@app.post("/api/v1/turn_off")
+def turn_off():
+    bulb.turn_off()
+    return bulb.turn_off()
 
+def discover_bulb():
+    bulbs = discover_bulbs()
+    for bulb in bulbs:
+        if bulb['capabilities']['name'] == 'Sam':
```

Comment:
nice! would be good to throw an exception if no bulb is found

Reply:
Done (I think?)